lib/OpenQA/Schema/ResultSet/DeveloperSessions.pm

```diff
@@ -22,7 +22,7 @@ sub register {
     my ($result, $worker_id, $is_session_already_existing) = $schema->txn_do(
         sub {
             # refuse if no worker assigned
-            my $worker = $schema->resultset('Workers')->search({job_id => $job_id})->first;
+            my $worker = $schema->resultset('Workers')->first({job_id => $job_id});
```

Comment:
Hm, the documentation says that `first` doesn't expect any arguments
https://metacpan.org/pod/DBIx::Class::ResultSet#first

Reply:
Right. That should be `find` here. Thx